src/app/components/DoctorCard/DoctorCard.tsx

```diff
@@ -43,7 +50,11 @@ export default function DoctorCard({
         </p>
       </div>
       {isShowButton && (
-        <ActionButton className={styles.actionButton} children="zum Profil" />
+        <ActionButton
+          className={styles.actionButton}
+          children="zum Profil"
+          onClickHandle={navigateToProfile}
+        />
```

Comment:
This should not be a button but rather a Link
https://reactrouter.com/web/api/Link

Reply:
Done!